src/media/css/account.styl

```diff
@@ -81,6 +81,10 @@
         display: none;
     }
 }
+.html-rtl .region span {
```

Comment:
You'd still follow each block wtih ```.html-rtl```, just in different files.

- The Fanatic of Organization Controversy